src/Native/System.Native/pal_process.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
```

Comment:
We should probably put this together with ForkAndExecProcess. I'm ok with the name pal_process, but maybe then rename pal_exec to pal_process.